README.md

```diff
@@ -144,28 +151,27 @@ SYNC Updating Policy: Test policy 1
 The command provides output to identify which actions have been taken. If using a Tyk Gateway, the Gateway will be
 automatically hot-reloaded.
 
-
 ## Example: Check the currently installed version of Tyk Sync
 
 To check the current Tyk Sync version, we need to run the version command:
 
-```
+```bash
 tyk-sync version
-v1.2.2
+v1.4.2
```

Comment:
Can we skip printing the version?
We will have to update it on each reason and I don't see much value in it

Reply:
sure, updated accordingly